src/components/ButtonPannel.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import '../index.css';
+import Button from './Button';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class ButtonPannel extends React.Component {
```

Comment:
- [ ] {OPTIONAL] Let's not overuse eslint-disable, if your component doesn't handle any state if should be a functional component (like App), not a class one.

[Check this article about stateful and stateless components for more on that .](https://programmingwithmosh.com/javascript/stateful-stateless-components-react/)